pkg/cmd/app_test.go

```diff
@@ -37,19 +36,9 @@ func TestGetEnvoyValidateArgs(t *testing.T) {
 		args        []string
 		expectedErr string
 	}{
-		{
-			name:        "--home-dir empty",
```

Comment:
removed empty tests because it is an odd use case, and it is hard to differentiate empty from never set in practice